db/schema.rb

```diff
@@ -113,6 +113,8 @@
     t.boolean "author_page", default: false
     t.string "desc"
     t.string "custom_path"
+    t.string "page_link"
+    t.integer "page_sort", default: 0
```

Comment:
I think columns in MySQL are numbered from 1, not 0, should the default be 1 here?

Reply:
Hmm not sure what you mean? This is a custom field, can be anything we want really no?

Reply:
The query would be

`SELECT `posts`.* FROM `posts` WHERE `posts`.`author_id` = 9729 AND `posts`.`author_page` = TRUE ORDER BY `posts`.`page_sort` ASC`

So it should be able to work with 0's since it just sorts them numerically

Reply:
Ahhhh you're right, don't know why I was replacing page_sort as if it was a variable